accessibility_monitoring_platform/apps/reports/views.py

```diff
@@ -322,3 +329,42 @@ class ReportWrapperUpdateView(UpdateView):
     def get_object(self, queryset=None):  # pylint: disable=unused-argument
         """Return report wrapper object"""
         return ReportWrapper.objects.all().first()
+
+
+class ReportVisitsMetricsView(ReportTemplateView):
+    """
+    View of list of published reports
+    """
+
+    template_name: str = "reports/report_visits_metrics.html"
+
+    def get_context_data(self, *args, **kwargs) -> Dict[str, Any]:
+        context: Dict[str, Any] = super().get_context_data(*args, **kwargs)
+        context["showing"] = self.request.GET.get("showing")
+        context["userhash"] = self.request.GET.get("userhash")
+
+        if context["userhash"]:
+            context["visit_logs"] = ReportVisitsMetrics.objects.filter(
```

Comment:
I see you've found the trick of assigning values directly into dictionaries to avoid having to enter typing information.